pyproject.toml

```
@@ -29,6 +29,7 @@ classifiers = [
 requires-python = ">=3.8"
 dependencies = [
     'certifi',
+    'python-dateutil==2.8.*',
```

Any major matching version should be okay
```suggestion
    'python-dateutil==2.*',
```